.github/workflows/linux-ci.yml

```diff
@@ -16,11 +16,21 @@ jobs:
 
     strategy:
       matrix:
-        version: [net6.0]
+        version: [net6.0, net7.0]
 
     steps:
     - uses: actions/checkout@v3
 
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '6.0.x'
```

Comment:
Do we need to install both 6.0 and 7.0 SDKs here?

Reply:
I copied this from last time: IIRC this was an issue with linux based images only (builds were failing without it). Did not spend time in researching it as it is temporary thing until images include .NET7.0